listenbrainz_spark/stats/tests/test_init.py

```diff
@@ -0,0 +1,39 @@
+	#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import datetime
+from listenbrainz_spark import utils, stats
+from pyspark.sql import Row
+from listenbrainz_spark.tests import SparkTestCase
+
+
+class InitTestCase(SparkTestCase):
+
+    def test_replace_days(self):
+        self.assertEqual(stats.replace_days(datetime.datetime(2019,5,12),13), datetime.datetime(2019,5,13))
+        
+    def test_adjust_months(self):
+        d1 = stats.adjust_months(datetime.datetime(2019,5,12),3,False)
+        d2 = datetime.datetime(2019,8,12)
+        self.assertEqual(d1, d2)
+        d1 = stats.adjust_months(datetime.datetime(2019,5,12),3)
+        d2 = datetime.datetime(2019,2,12)
+        self.assertEqual(d1, d2)
+        
+    def test_adjust_days(self):
+        d1 = stats.adjust_days(datetime.datetime(2019,5,12), 3, False)
+        d2 = datetime.datetime(2019,5,15)
+        self.assertEqual(d1, d2)
+        d1 = stats.adjust_days(datetime.datetime(2019,5,12), 3)
+        d2 = datetime.datetime(2019,5,9)
+        self.assertEqual(d1, d2)
+        
+        
+    def test_run_query(self):
+        df = utils.create_dataframe(Row(column1=1, column2=2), schema=None)
+        utils.register_dataframe(df,'table')
+        new_df = stats.run_query("""
+                               SELECT *
+                                FROM table
+                               """)
```

Comment:
Did you read the guide on writing SQL queries, if not please go through it?
https://github.com/metabrainz/listenbrainz-labs/blob/master/listenbrainz_spark/sql/create_dataframes_queries.py#L84
You may refer to this link as well. 
There are too many tabs in this query, also the query should be right-aligned.

Reply:
@vansika any idea on how to right align in Spyder editor?

Reply:
Do it by hand. I haven't found an editor that supports automatically indenting SQL to our conventions.

Reply:
new_df = stats.run_query("""
         SELECT *
          FROM table
          """)

Reply:
Is this fine?